tests/acceptance/features/webUIFilesActionMenu/versions.feature

```diff
@@ -85,3 +85,14 @@ Feature: Versions of a file
     And the user uploads overwriting file "lorem.txt" using the webUI
     And the user browses to display the "versions" details of file "lorem.txt"
     Then the versions list should contain 1 entries
+
+   @issue-ocis-1328 @disablePreviews
+  Scenario: sharee can see the versions of a file
```

Comment:
Pretty sure this one was deleted as flaky(?) also failes CI in this PR

Reply:
Yes. It was skipped because of its flakiness but should it be entirely removed? I'll try to find out if it's the test timing problem or the backend issue. if it's the latter we can skip it again?